test/js/bun/udp/udp.test.ts

```diff
@@ -0,0 +1,488 @@
+import { udpSocket } from "bun";
```

Comment:
This is going to be annoying, but can we break up the tests into 2 places:

`test/js/bun/udp` and `test/js/node/dgram`